concept/type/impl/ThingTypeImpl.java

```diff
@@ -329,11 +329,12 @@ private void ownsAttribute(AttributeTypeImpl attributeType) {
 
         TypeVertex attVertex = attributeType.vertex;
         TypeEdge existingEdge;
-        if ((existingEdge = vertex.outs().edge(OWNS_KEY, attVertex)) != null ||
-                (existingEdge = vertex.outs().edge(OWNS, attVertex)) != null) {
-            existingEdge.delete();
+        if ((existingEdge = vertex.outs().edge(OWNS, attVertex)) != null) {
+            existingEdge.unsetOverridden();
```

Comment:
if this exact edge already exists, we just unset the overridden vertex. 
If it is the wrong type of edge (eg. OWNS_KEY), then delete it before creating it. If there is no edge, we just create it immediately